app/webpack/javascripts/modules/external_users/claims/FeeTypeCtrl.js

```diff
@@ -0,0 +1,69 @@
+moj.Modules.FeeTypeCtrl = {
```

Comment:
Probably need some tests around this javascript?, but I think this is what @naseberry is already working on.

Reply:
yes, he is